pkg/types/conf_test.go

```diff
@@ -116,6 +116,12 @@ var _ = Describe("config operations", func() {
 			"type": "multus",
 			"logLevel": "debug",
 			"logFile": "/var/log/multus.log",
+			"logOptions": {
+			    "maxAge": 5,
+                "maxSize": 100,
+                "maxBackups": 5,
+                "compress": true
+			}
```

Comment:
it seems to be different indent. Did you do `gofmt -s -w` command to fix golang format?

Reply:
Sure(little strange), I did fix the golang format, but it seems to not work well